src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,169 @@
+package duke;
+
+import exceptions.DukeException;
+
+import java.util.Scanner;
+
+public class Duke {
+    private static Task[] taskList = new Task [100];
+    static int taskIndex=0;
+
+    public static void main(String[] args) {
+
+        greetUser("hi");
+        startChatbot();
+        greetUser("bye");
+    }
+
+    private static void startChatbot() {
+        Scanner input = new Scanner(System.in);
+        String task = "";
+        while(true) {
+            try {
+                task = input.nextLine();
+                System.out.println("    ____________________________________");
+                if (task.equals("bye")) {
+                    break;
+                } else if (task.equals("list")) {
+                    listTasks();
+                } else if (task.startsWith("mark ")) {
+                    markTask(task);
+                } else if (task.startsWith("unmark ")) {
+                    unmarkTask(task);
+                } else if (task.startsWith("todo") || task.startsWith("deadline") || task.startsWith("event")) {
+                    addTask(task);
+                } else {
+                    throw new DukeException();
+                }
+                System.out.println("    ____________________________________");
+            } catch (DukeException e) {
+                System.out.println("    Sorry I do not know what that means!");
+            }
+
+        }
+    }
+
+    private static void greetUser(String greeting) {
+        if(greeting.equals("hi")) {
+            String logo = " ____        _\n"
+                    + "|  _ \\ _   _| | _____\n"
+                    + "| | | | | | | |/ / _ \\\n"
+                    + "| |_| | |_| |   <  __/\n"
+                    + "|____/ \\__,_|_|\\_\\___|\n";
+            System.out.println(logo);
+            System.out.println("    ____________________________________");
+            System.out.println("    Hello! I'm Duke");
+            System.out.println("    What can I do for you?\n");
+        }
+        else {
+            System.out.println("    ____________________________________");
+            System.out.println("    Bye. Hope to see you again soon!");
+            System.out.println("    ____________________________________");
+        }
+    }
+
+    public static void listTasks() {
+        System.out.println("    Here are the tasks in your list:");
+        for(int i=0;i< taskList.length;i++) {
```

Comment:
According to the Java coding standard, the for loop should resemble the format (take note of the white spaces): 
```
for (i = 0; i < 100; i++) {
    sum += value[i];
}
```